dev-docs/research/fanout-2026-04-11-linearizability-wave1/prompts/README.md

```
@@ -0,0 +1,20 @@
+> **Status:** Active
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)